usecases/doc.go

```diff
@@ -0,0 +1,80 @@
+package usecases
```

Comment:
this package name is not good, you could have a docs package with the docs stuff, a rpc package and a request one